pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -1382,7 +1382,7 @@ var _ = Describe("Validating Webhook", func() {
 
 				vmi.Spec.Networks = append(vmi.Spec.Networks,
 					v1.Network{Name: networkName, NetworkSource: v1.NetworkSource{
-						Multus: &v1.CniNetwork{NetworkName: networkName}}})
+						Npwgv1: &v1.CniNetwork{NetworkName: networkName}}})
```

Comment:
There should be tests, unit and functional, that still use ```multus``` attribute, to make sure we don't break old manifests.